dashboard/app/views/levels/_apps_dependencies.html.haml

```diff
@@ -24,7 +24,8 @@
 
 -# Droplet script dependencies (all editCode - regardless of editor)
 - if use_droplet
-  %script{src: asset_path('js/jsinterpreter/acorn_interpreter.js')}
+  %script{src: minifiable_asset_path('js/jsinterpreter/acorn.js')}
+  %script{src: minifiable_asset_path('js/jsinterpreter/interpreter.js')}
```

Comment:
The `minifiable_asset_path` function means that we'll use the non-dev mode files (appended with .min.js) normally, but will use the dev-mode files (appended with just .js) when `Rails.configuration.pretty_sharedjs` is set. This is how our other apps code is served from the dashboard.